wifiphisher.py

```diff
@@ -978,6 +962,120 @@ def get_hostapd():
     channel, essid, ap_mac = copy_AP()
     hop_daemon_running = False
 
+    # get all the local templates
+    local_templates = phishingpage.get_local_templates()
+
+    # loop through the local templates
+    for name in local_templates:
+        # add the template to the database
+        phishingpage.add_template(name)
+
+    # get template_database
+    template_database = phishingpage.get_template_database()
+
+    # get all the templtes names for display
+    template_names = list(template_database.keys())
+
+    # variable used to indicate correct template selection
+    template_selected = False
+
+    # loop until all operations for template selection is done
+    while True:
+        # check if the template argument is set and is correct
+        if args.template and phishingpage.check_template(args.template):
+            # set the template name
+            template_name = args.template
+
+            # exit loop to avoid interactive selection
+            break
+        # in case of an invalid template
+        elif args.template and not phishingpage.check_template(args.template):
+            raise InvalidTemplate
+
+        # loop until a valid template is selected
+        while not template_selected:
+            # display start of template names
+            print "\nList Of All Templates: \n"
+
+            # display the templates
+            for number in range(len(template_names)):
+                print ("[" + G + str(number + 1) + W + "] " +
+                       template_names[number])
+
+            # get user's choice
+            choosen_template = raw_input("Please select one of the above "
+                                         "templates: ")
```

Comment:
You should follow the format we are using when choosing an AP. See <a href="https://github.com/sophron/wifiphisher/blob/master/wifiphisher.py#L456">here</a>.